lib/homePage.dart

```diff
@@ -51,6 +53,19 @@ class HomePageView extends State<HomePage> with SingleTickerProviderStateMixin {
     _animationController?.stop();
     _animationController?.dispose();
     super.dispose();
+    WidgetsBinding.instance.removeObserver(this);
+  }
+
+  @override
+  void didChangeAppLifecycleState(AppLifecycleState state) {
```

Comment:
just above the function name so that they can understand what it is

Reply:
Ok, I will add the comments, Thanks for the suggestion.